shared/src/main/scala/com/cibo/evilplot/plot/components/Axes.scala

```diff
@@ -213,30 +217,132 @@ object Axes {
   trait AxesImplicits {
     protected val plot: Plot
 
+    /** Add a continuous axis to the plot.
+      * @param boundsFn         Takes a plot and returns the bounds this axis will display.
+      * @param position         The side of the plot to add the axis.
+      * @param tickCount        The number of tick lines.
+      * @param tickRenderer     Function to draw a tick line/label.
+      * @param labelFormatter   Custom function to format tick labels.
+      * @param tickCountRange   Allow searching over axis labels with this many ticks.
+      * @param updatePlotBounds Set plot bounds to match the axis bounds.
+      * @param fixedBounds      Force ticks to match bounds.
+      */
+    def continuousAxis(
+      boundsFn: Plot => Bounds,
+      position: Position,
+      tickCount: Option[Int] = None,
+      tickRenderer: Option[TickRenderer] = None,
+      labelFormatter: Option[Double => String] = None,
+      tickCountRange: Option[Seq[Int]] = None,
+      updatePlotBounds: Boolean = true,
+      fixedBounds: Boolean = true
+    )(implicit theme: Theme): Plot = {
+      val defaultRotation = if (position == Position.Bottom || position == Position.Top) {
+        theme.elements.categoricalXAxisLabelOrientation
```

Comment:
This is the continuous function but these theme values are for categorical axes.